test/unit/cross_lane_ops_test/test/blend_unpack_byte_hi.cpp

```diff
@@ -57,24 +57,12 @@ namespace rocwmma
         static inline std::vector<ThreadBlockT> threadBlocks()
         {
             auto warpSize = HipDevice::instance()->warpSize();
-            // clang-format off
-            return {
-                        {warpSize, 1},
-                        {warpSize * 2, 1},
-                        {warpSize * 4, 1}
-                    };
-            // clang-format on
+            return {{warpSize, 1}};
         }
 
         static inline std::vector<ProblemSizeT> problemSizes()
         {
-            // clang-format off
-            return {
-                        {64, 64},
-                        {128, 128},
-                        {256, 256}
-                    };
-            // clang-format on
+            return {{warpSize, 1}};
```

Comment:
Need to get warpSize from device

Reply:
Done. The `warpSize` is also a const value defined by hip.